util/Migrator/DbScripts/2022-07-20_00_CreateSecretTable.sql

```diff
@@ -0,0 +1,22 @@
+IF OBJECT_ID('[dbo].[Secret]') IS NULL
+BEGIN
+	CREATE TABLE [dbo].[Secret] (
+		[Id] UNIQUEIDENTIFIER NOT NULL,
+		[OrganizationId] UNIQUEIDENTIFIER NOT NULL,
+        [Key] NVARCHAR(MAX) NULL,
```

Comment:
We seem to be mixing spaces and tabs in this file. Please ensure it uses spaces.